cmd/session-resume.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/minio/pkg/probe"
+	"github.com/minio/minio/pkg/trie"
+)
+
+var sessionResume = cli.Command{
+	Name:   "resume",
+	Usage:  "Resume a previously saved session.",
+	Action: mainSessionResume,
+	Flags:  globalFlags,
+	Before: setGlobalsFromContext,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} [SESSION-ID/all]
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Resume session. 
+	 $ {{.HelpName}} ygVIpSJs
```

Comment:
Can you open a bug if possible to support resume all we don't need it in this PR.